src/web/views/Components/Slider/Slider.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <main class="article slider">
+    <div class="article-wrapper">
+      <BIMDataText component="h1" color="color-primary">
+        {{ $route.name }}
+      </BIMDataText>
+      <ComponentCode :componentTitle="$route.name" language="javascript">
+        <template #module>
+          <div class="m-t-24">
+            <BIMDataSlider
+              v-model="slider"
+              :min="minRange"
+              :max="maxRange"
+              :step="step"
+              :tooltip="tooltip"
+            />
+          </div>
+        </template>
+
+        <template #parameters>
+          <BIMDataInput
+            type="number"
+            placeholder="Min range"
+            v-model="minRange"
+          />
+          <BIMDataInput
+            type="number"
+            placeholder="Max range"
+            v-model="maxRange"
+          />
+          <BIMDataInput
+            type="number"
+            placeholder="Slider default value"
+            v-model="slider"
```

Comment:
You should decompse `v-model="slider"` this way ton avoid warnings in the console:
```
:modelValue="slider"
@update:modelValue="slider = +$event"
```